opendebates/settings.py

```diff
@@ -8,6 +8,31 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 FIXTURE_DIRS = [os.path.join(BASE_DIR, 'fixtures')]
 
+SITE_ID = 1
+SITE_DOMAIN = os.environ.get("SITE_DOMAIN", "127.0.0.1:8000")
```

Comment:
Suggest looking at `deployment/templates/local_settings.py` and keying off of `{{ environment }}`.